community/kernel/src/main/java/org/neo4j/graphdb/schema/ConstraintDefinition.java

```diff
@@ -20,22 +20,43 @@
 package org.neo4j.graphdb.schema;
 
 import org.neo4j.graphdb.Label;
+import org.neo4j.graphdb.RelationshipType;
 
 /**
  * Definition of a constraint.
+ *
+ * <b>Note:</b> This interface is going to be changed/removed in next major release to better cope with node and
+ * relationship constraints which are quite different concepts.
  */
 public interface ConstraintDefinition
 {
     /**
+     * This accessor method returns a label which this constraint is associated with iff this constraint has type
```

Comment:
Intention was to mean https://en.wikipedia.org/wiki/If_and_only_if. This is probably confusing.